### DISCUSSION TITLE
Apple Silicon

### DISCUSSION THREAD

**Discussion by ghost** [wxFormBuilder/wxFormBuilder#755] [General]
Any chance of add an apple silicon build (or universal) to the released binary downloads?

**Comment by sodevel** [comment]
Short answer: No

Long answer:
All releases are built by GitHub CI, i can only release what this system can build. Right now, there are only Intel Runners available, so the only way to get Apple Silicion (or Universal) binaries is to cross-compile. wxFB uses CMake so it should be possible to cross-compile wxFB itself, however i have no experience how to setup this. But wxFB depends on libraries (mostly wxWidgets, Boost is used Header-Only currently) supplied by Homebrew, and Homebrew does not support Universal binaries or cross-compiling. There would need to be used a different approach to get wxWidgets. I don't have access to a Mac, i have only GitHub CI, and using only the CI to figure out such a complicated issue is WAY to cumbersome and time consuming (i remember all the pain i had to get the Mac OS build going at all).

TL;DR: I won't be able to setup a cross-compiling environment. GitHub has Apple Silicon Runners on its Roadmap, the best bet is to wait until they are released. With such Runners available, most probably i have just to change one line in the Workflow to also get Apple Silicon Binaries getting build.
